scripts/init_dev

```diff
@@ -38,10 +38,15 @@ sudo apt-get update -q
 # sudo apt-get -y upgrade
 
 # Install pip and couchdb (couchdb is not available via rosdep -- see below)
-echo "Install pip, couchdb and boostrapping tools"
-sudo apt-get install -y -q python-pip couchdb
+echo "Install pip, couchdb and bootstrapping tools"
+sudo apt-get install -y -q couchdb
+sudo easy_install pip
```

Comment:
@rwdavis513 would this also solve the `Requests` versioning conflict we have with `apt-get`'s version of `pip`?